server/config.cpp

```diff
@@ -505,6 +508,10 @@ int getPkillInCombatDisabled() {
     return(gConfig->pkillInCombatDisabled);
 }
 
+int getPlayersCanMudmailStaff() {
```

Comment:
Move this into Config and make the variable it's pulling private.

Reply:
This i do not understand. Probably due to rust and cobwebs in my head about cpp.
 Is the pk in combat one the same way?

Reply:
The pkill in combat is a bad example :-/